src/unix/lwt_unix.cppo.mli

```diff
@@ -298,6 +298,17 @@ val read : file_descr -> bytes -> int -> int -> int Lwt.t
     except [Unix.Unix_error Unix.EAGAIN], [Unix.Unix_error Unix.EWOULDBLOCK] or
     [Unix.Unix_error Unix.EINTR]. *)
 
+val pread : file_descr -> bytes -> file_offset:int -> int -> int -> int Lwt.t
+(** [pread fd buf ~file_offset ofs len] on file descriptors allowing seek,
+    reads up to [len] bytes from [fd] at offset [file_offset] from the
+    beginning of the file, and writes them to [buf], starting at offset [ofs].
+
+    The current position of the file descriptor is undefined when [pread]
+    returns.
```

Comment:
Can you change this to positively state what the behavior is on Unix and on Windows?

Reply:
Of course